com.realm667.WolfenDoom_Blade_of_Agony.yaml

```diff
@@ -93,7 +100,7 @@ modules:
     - install -Dm 644 gzdoom.sf2 /app/share/sounds/sf2/gzdoom.sf2
   - type: script
     commands:
-    - gzdoom -file boa_c2.pk3 +fluid_patchset /app/share/sounds/sf2/gzdoom.sf2 $@
+    - gzdoom -file boa_c2.pk3 -file boa_c2_fix_37.pk3 +fluid_patchset /app/share/sounds/sf2/gzdoom.sf2 $@
```

Comment:
Multiple `-file`s is wrong

This is correct:
`gzdoom -file boa_c2.pk3 boa_c2_fix_37.pk3 +fluid_patchset /app/share/sounds/sf2/gzdoom.sf2 $@`